src/jdk.compiler/share/classes/com/sun/tools/javac/resources/javac.properties

```diff
@@ -125,8 +126,6 @@ javac.opt.arg.profile=\
     <profile>
 javac.opt.arg.release=\
     <release>
-javac.opt.arg.release=\
-    <release>
```

Comment:
I assume this is just an incidental cleanup.

Reply:
Yes, note it was a duplicated resource, that was flagged by the IDE.